[Go] feat: update docs for dd-trace-go/v2 (#25897)
DataDog/documentation: content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/go.md

  text: "Explore your services, resources and traces"
---

<div class="alert alert-info">

> [PREVIEW] The preview version of Go Tracer v2 has been released! See <a href="/content/en/tracing/trace_collection/custom_instrumentation/go/migration.md">our migration guide</a> to upgrade.

Reviewer: ```suggestion
The Go Tracer v2 is now in Preview! See <a href="/tracing/trace_collection/custom_instrumentation/go/migration">our migration guide</a> to upgrade.
```
